src/domain_knowledge.jl

```diff
@@ -31,6 +31,11 @@ This is within the policy when a RolloutEstimator is passed to an AbstractMCTSSo
 mutable struct SolvedRolloutEstimator{P<:Policy, RNG<:AbstractRNG}
     policy::P
     rng::RNG
+    depth::Union{Int, Nothing}
```

Comment:
The end-user doesn't directly work with `SolvedRolloutEstimator`. Rather, the user instantiates a `RolloutEstimator` object  with all necessary options and puts it in the `estimate_value` field for MCTS (or any other applicable solver).

I would recommend giving the `RolloutEstimator` struct a `max_depth` field that then fills the corresponding field in `SolvedRolloutEstimator` in the `convert_estimator` call.

Reply:
I think `eps` could also be added, with default <1.

Reply:
You're right. I think a reasonable middle ground between infinite rollouts and a low default `max_depth` would be an epsilon field that defaults to something like 0.01. In this case, a pomdp with discount 0.99 would run for 459 steps which is high but not unreasonable.